src/20240108_AaveV2Polygon_ReserveFactorUpdatesJan152024/AaveV2Polygon_ReserveFactorUpdatesJan152024_20240108.sol

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.19;
```

Comment:
```suggestion
pragma solidity ^0.8.0;
```

@efecarranza Can we not make the solc version strict, as we specify the solc version globally on the `foundry.toml`.